functorch/functorch/_src/partitioners.py

```diff
@@ -164,6 +165,76 @@ def default_partition(
 
     return _extract_fwd_bwd_modules(joint_module, saved_values)
 
+def move_input_mutations_into_submodule(fx_g: fx.GraphModule):
+    mutable_nodes: List[Node] = []  # the copy_() and as_strided_() nodes, that we'll delete move to a submodule.
+    node_map: Dict[Node, Node] = {}
+    node_to_placeholder: Dict[Node, Node] = OrderedDict()  # mapping of nodes from old graph to placeholder in new graph
+    mutation_subgraph = fx.Graph()
+
+    def remap_inputs(x):
+        if x.op == "get_attr":
+            pass
```

Comment:
how come?

Reply:
yep this was me misunderstanding FX nodes again, removing